spec/factories/tailoring.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :v2_tailoring, class: 'V2::Tailoring' do
+    profile do
+      V2::Profile
+        .joins(:os_minor_versions)
+        .find_by!(
+          ref_id: ref_id,
+          profile_os_minor_versions: {
+            os_minor_version: os_minor_version
+          }
+        )
+    end
+    value_overrides { profile.value_overrides }
+
+    transient do
+      ref_id { policy.profile.ref_id }
+      os_major_version { 7 }
+    end
+
+    trait :with_tailored_values do
+      value_overrides do
+        profile.value_overrides&.each do |k, v|
+          # overriding (tailoring) original values
+          profile.value_overrides[k] = v + SecureRandom.random_number(100)
+        end
+      end
+    end
```

Comment:
Same as above, randomization of the right side isn't enough, it should be also just overriding a few values + shuffle in some from the security guide (follow what happens with rules).